pages/home.jsx

```diff
@@ -101,13 +105,15 @@ var BlogSection = React.createClass({
     }
   },
   componentDidMount: function() {
-    var self = this;
-    blogFeedLoader(this, window, "https://blog.webmaker.org/tag/teachtheweb/feed", function(data) {
-      self.setState({
-        featuredPostData: data.featuredPostData,
-        latestPostsData: data.latestPostsData
+    this.props.loadBlogPosts(function(data) {
```

Comment:
Oh, and also, it's kind of specific to the Google Feed API implementation--when we move the implementation to the Teach API, it won't take a feed URL parameter, it'll just be built-in to the Teach API.